content/applications/finance/accounting/vendor_bills/deferred_expenses.rst

```diff
@@ -1,195 +1,176 @@
 
-.. image:: deferred_expenses/deferred_expenses04.png
-   :align: center
-   :alt: Modification of a posted journal item's account in Odoo Accounting
+One entry, dated on the same day as the bill, moves the bill amounts from the expense account to
+the deferred account. The other entries are deferral entries which will, month after month, move the
+bill amounts from the deferred account to the expense account to recognize the expense.
 
-Deferred Expenses entries
-=========================
+.. example::
+   You can defer a January bill of $1200 over 12 months by specifying a start date of 01/01/2023
+   and an end date of 12/31/2023. At the end of August, $800 is recognized as an expense,
+   whereas $400 remains on the deferred account.
 
-Create a new entry
-------------------
+Reporting
+=========
 
-A **Deferred Expense entry** automatically generates all journal entries in *draft mode*. They are
-then posted one by one at the right time until the full amount of the expense is recognized.
+The deferred expense report computes an overview of the necessary deferral entries for each account.
+To access it, go to :menuselection:`Accounting --> Reporting --> Deferred Expense`.
 
-To create a new entry, go to :menuselection:`Accounting --> Accounting --> Deferred Expense`, click
-on *Create*, and fill out the form.
+To view the journal items of each account, click on the account name and then :guilabel:`Journal
+Items`.
 
-Click on **select related purchases** to link an existing journal item to this new entry. Some
-fields are then automatically filled out, and the journal item is now listed under the **Related
-Expenses** tab.
+.. image:: deferred_expenses/deferred_expense_report.png
+   :alt: Deferred expense report
 
-.. image:: deferred_expenses/deferred_expenses05.png
-   :align: center
-   :alt: Deferred Expense entry in Odoo Accounting
-
-Once done, you can click on *Compute Deferral* (next to the *Confirm* button) to generate all the
-values of the **Expense Board**. This board shows you all the entries that Odoo will post to
-recognize your expense, and at which date.
+.. note::
+    Only bills whose accounting date is before the end of the period of the report
+    are taken into account.
 
-.. image:: deferred_expenses/deferred_expenses06.png
-   :align: center
-   :alt: Expense Board in Odoo Accounting
+.. _vendor_bills/deferred/generate_manually:
 
-What does "Prorata Temporis" mean?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Generate grouped deferral entries manually
+==========================================
 
-The **Prorata Temporis** feature is useful to recognize your expense the most accurately possible.
+If you have a lot of deferred revenues and wish to reduce the number of journal entries created, you
+can generate deferral entries manually. To do so, set the :guilabel:`Generate Entries` field in the
+**Settings** to :guilabel:`Manually & Grouped`. Odoo then aggregates the deferred amounts in a
+single entry.
```

Comment:
same as in the other file: I used the contents of the tip that you had at the end of the file here instead. I felt is was a) not really a tip and b) more useful here. I also removed the "example" part of the tip as I felt it wasn't necessary.